README.md

```diff
@@ -38,61 +47,108 @@ The following screenshots give a close insight into KubeSphere. Please check [Wh
 
 ## Demo Environment
 
-Using the account `demo1 / Demo123` to log in the [demo environment](https://demo.kubesphere.io/). Please note the account is granted view access. You can also have a quick view of [KubeSphere Demo Video](https://youtu.be/u5lQvhi_Xlc).
+🎮 Using the account `demo1 / Demo123` to log in the [demo environment](https://demo.kubesphere.io/). Please note the account is granted view access. 
 
-## Architecture
+🖥 You can also have a quick view of [Demo video](https://youtu.be/YxZ1YUv0CYs).
 
-KubeSphere uses a loosely-coupled architecture that separates the [frontend](https://github.com/kubesphere/console) from the [backend](https://github.com/kubesphere/kubesphere). External systems can access the components of the backend which are delivered as Docker containers through the REST APIs. See [Architecture](https://kubesphere.io/docs/introduction/architecture/) for details.
+## Features
 
-![Architecture](docs/images/architecture.png)
+<details>
+  <summary><b>🕸 Provisioning Kubernetes Cluster</b></summary>
+  Support deploy Kubernetes on any infrastructure, support online and air-gapped installation, <a href="https://kubesphere.io/docs/installing-on-linux/introduction/intro/">learn more</a>.
+  </details>
+
+<details>
+  <summary><b>🔗 Kubernetes Multi-cluster Management</b></summary>
+  Provide a centralized control plane to manage multiple Kubernetes clusters, support propagate an app to multiple K8s clusters across different cloud providers.
+  </details>
+
+<details>
+  <summary><b>🤖 Kubernetes DevOps</b></summary>
+  Provide out-of-box CI/CD based on Jenkins, and offers automated workflow tools including binary-to-image (B2I) and source-to-image (S2I), <a href="https://kubesphere.io/devops/">learn more</a>.
```

Comment:
I'm wondering if it's ok to change the DevOps link to https://github.com/kubesphere/ks-devops

For developers, might want to check it git repository instead.

Reply:
I am going to add a separate table that will include the major GitHub repositories at the bottom of README. The [website](https://kubesphere.io/devops/) is more informative for a newbie, we can add the git repository to it if you want.